addons/survey/models/survey_question.py

```diff
@@ -98,6 +98,7 @@ def default_get(self, fields):
         ('12', '1'), ('6', '2'), ('4', '3'), ('3', '4'), ('2', '6')],
         string='Number of columns', default='12',
         help='These options refer to col-xx-[12|6|4|3|2] classes in Bootstrap for dropdown-based simple and multiple choice questions.')
+    answer_image = fields.Boolean(default=False)
```

Comment:
Rename to allow_answer_image to make it sound like it's a Boolean.

Always put the name of the field here, and then override in the view if needed.
Also, we try to fit the field label in a single line if possible, so "Allow images on answers" is a bit too long.
Finally, no need to specify "default=False", it's the default behavior.

I would go for:
`allow_answer_image = fields.Boolean("Images on answers")`

Reply:
done